.ci/jenkins/Jenkinsfile.kie-kogito-post-release

```
@@ -0,0 +1,205 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.jenkinsci.plugins.workflow.libs.Library
+import org.kie.jenkins.MavenCommand
+
+docsRepo = "kie-docs"
+COMMIT_MSG = "Bumped up kie-kogito-docs version"
+
+
+pipeline {
+
+    agent {
+        label 'kie-rhel8 && !built-in'
+    }
+
+    options{
+        timestamps()
+        timeout(time: 60, unit: 'MINUTES')
+    }
+
+    tools {
+        maven "${BUILD_MAVEN_TOOL}"
+        jdk "${BUILD_JDK_TOOL}"
+    }
+
+    environment {
+        // Static env is defined into .jenkins/dsl/jobs.groovy file
+        KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
+        PR_BRANCH = "bump-up-kie-kogito-docs-${util.generateHash(10)}"
+    }
+
+    stages{
+        stage('CleanWorkspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Initialize') {
+            steps{
+                script{
+                    sh 'printenv'
+                    dir(docsRepo){
+                        checkoutRepo(docsRepo, GIT_AUTHOR, getKogitoDocsBranch(), GIT_AUTHOR_CREDENTIALS_ID)
+                    }
+                }
+            }
+        }
+        stage('Create pull request branch') {
+            steps{
+                script{
+                    dir(docsRepo){
+                        githubscm.createBranch(PR_BRANCH)
+
+                    }
+                }
+            }
+        }
+        stage('Update kogito-docs version in kie-docs') {
+            steps {
+                script {
+                    dir("${docsRepo}/doc-content/kogito-docs") {
+                        maven.mvnVersionsSet(parseVersion(getVersion()))
+                        githubscm.commitChanges(COMMIT_MSG)
+                    }
+                }
+            }
+        }
+        stage('Build kie-docs/doc-content/kogito-docs') {
+            steps {
+                configFileProvider([configFile(fileId: '771ff52a-a8b4-40e6-9b22-d54c7314aa1e', targetLocation: 'jenkins-settings.xml', variable: 'SETTINGS_XML_FILE')]) {
+                    dir("${docsRepo}/doc-content/kogito-docs") {
+                    sh""" mvn clean install -Dfull -s $SETTINGS_XML_FILE -Dkie.maven.settings.custom=$SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true
+                         # clean files created by the build
+                         git clean -d -f """
```

Reformat a bit and used `"` for the `$SETTINGS_XML_FILE`, in case there is any whitespace into the path
```suggestion
                    sh """
                        mvn clean install -Dfull -s "$SETTINGS_XML_FILE" -Dkie.maven.settings.custom="$SETTINGS_XML_FILE" -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true
                             # clean files created by the build
                             git clean -d -f 
                         """
```